README.md

```diff
@@ -250,82 +255,49 @@ useEffect(() => {
 </activity>
 ```
 
-### (OPTIONAL) Setup #3: Support Magic Link Redirects
-
-Supporting Magic Link authentication in flows requires some platform specific setup:
-
-- On Android: add another path entry to the [App Links](https://developer.android.com/training/app-links#android-app-links).
-  This is essentially another path in the same as the app link from the [previous setup step](#setup-2-enable-app-links),
-  with different handling logic. Refer to the previous section for the manifest setup.
-- On iOS: You'll need to [support associated domains](https://developer.apple.com/documentation/xcode/supporting-associated-domains?language=swift).
-
-Regardless of the platform, another path is required to handle magic link redirects specifically. For the sake of this README, let's name
-it `/magiclink`
-
-#### Add the required Linking logic
-
-_this code example demonstrates how app links or universal links should be handled - you can customize it to fit your app_
-
-```js
-import { useFlow } from '@descope/react-native-sdk'
-
-const flow = useFlow()
-
-useEffect(() => {
-  Linking.addEventListener('url', async (event) => {
-    if (event.url === 'my-deep-link-for-authenticating') {
-      try {
-        await flow.exchange(event.url) // Call exchange to complete the flow
-      } catch (e) {
-        // Handle errors here
-      }
-    } else if (event.url === 'my-deep-link-for-magic-links') {
-      // Adding the magic link handling here
-      try {
-        await flow.resume(event.url) // Resume the flow after returning from a magic link
-      } catch (e) {
-        // Handle errors here
-      }
-    }
-  })
-  return () => {
-    Linking.removeAllListeners('url')
-  }
-}, [flow])
-```
-
 ### Run a Flow
 
 After completing the prerequisite steps, it is now possible to run a flow.
-The flow will run in a [Custom Tab](https://developer.chrome.com/docs/android/custom-tabs/) on Android,
-or via [ASWebAuthenticationSession](https://developer.apple.com/documentation/authenticationservices/aswebauthenticationsession) on iOS.
-Run the flow by calling the flow start function:
+The flow will run in a dedicated `FlowView` component which receives `FlowOptions`.
+The `FlowOptions` defines all of the options available when running a flow on both
+Android and iOS. Read the component documentation for a detailed explanation.
 
 ```js
-import { useFlow } from '@descope/react-native-sdk'
```

Comment:
just a question - do we still want to document the old `useFlow` way?

Reply:
We don't on other platforms. This is to be phased out, but the docs are still available in the repo history.